test/parallel/test-module-print-timing.mjs

```diff
@@ -0,0 +1,91 @@
+import { spawnPromisified } from '../common/index.mjs';
+import tmpdir from '../common/tmpdir.js';
+import assert from 'node:assert';
+import { readFile } from 'node:fs/promises';
+import { execPath } from 'node:process';
+import { it } from 'node:test';
+
+tmpdir.refresh();
+
+it('should print the timing information for cjs', async () => {
+  const result = await spawnPromisified(execPath, ['--eval', 'require("url");'], {
+    env: {
+      NODE_DEBUG: 'module_timer',
+    },
+  });
+
+  assert.strictEqual(result.code, 0);
```

Comment:
You might want to log the stdout and stderr instead of going straight to asserting the exit code for getting more information out of the CI, or use a utility like `spawnSyncAndAssert` in `common/child_process` which is designed to log more things in the CI on mismatch.

Reply:
I'm getting the following error:

```
 /home/iojs/build/workspace/node-test-commit-linux-containered/out/Release/node: error while loading shared libraries: libcrypto.so.81.3: cannot open shared object file: No such file or directory
22:44:32     
22:44:32     [process 1608929]: --- stdout ---
22:44:32     
22:44:32     [process 1608929]: status = 127, signal = null
22:44:32     [process 1608953]: --- stderr ---
22:44:32     /home/iojs/build/workspace/node-test-commit-linux-containered/out/Release/node: error while loading shared libraries: libcrypto.so.81.3: cannot open shared object file: No such file or directory
22:44:32     
22:44:32     [process 1608953]: --- stdout ---
```

I tried to change the `execPath` but it didn't seems to change the error :/

Reply:
Does the tracing stuff rely on crypto somehow? There might be something wrong with the trace events in shared OpenSSL mode. You can try skipping tracing in those builds.

Reply:
Don't replace env -- you'll lose `LD_LIBRARY_PATH` which will prevent the OpenSSL libraries (or any of the shared libs for [node-test-commit-linux-containered](https://ci.nodejs.org/job/node-test-commit-linux-containered/)) from being found. Extend it instead (add/removing as necessary).